snap_stats.js

```diff
@@ -97,10 +98,19 @@ mergeObjectsKeys(snapsByUser, messagesByUser);
 addFields(snapsByUser, { sent: 0, received: 0 });
 addFields(messagesByUser, { sent: 0, received: 0 });
 
-const users = Object.keys(snapsByUser);
-const totalSnaps = users.map(user => snapsByUser[user].sent + snapsByUser[user].received);
-const messagesCount = users.map(user => messagesByUser[user].sent + messagesByUser[user].received);
-const names = users.map(u => nameByUser[u] || u);
+var users = Object.keys(snapsByUser);
+var totalSnaps = users.map(user => snapsByUser[user].sent + snapsByUser[user].received);
+var messagesCount = users.map(user => messagesByUser[user].sent + messagesByUser[user].received);
+var names = users.map(u => nameByUser[u] || u);
```

Comment:
Note that this project uses eslint. To make your compliant `yarn install` then `./node_modules/.bin/eslint snap_stats.js`